### ISSUE TITLE
select.isOptionDisabled is not a function

### ISSUE THREAD

**Issue by rob-forager** [thisbeyond/solid-select#28]
When using the latest version of solid-select (0.13.0) and the latest version of Solid (1.6.0) + Vite (3.1.8), I get this stack trace when clicking on the component:

```
dev.js:943 Uncaught TypeError: select.isOptionDisabled is not a function
    at get isDisabled [as isDisabled] (select.jsx:27:50)
    at Option._$effect._v$11 [as fn] (select.jsx:109:67)
    at runComputation (dev.js:693:22)
    at updateComputation (dev.js:676:3)
    at createRenderEffect (dev.js:205:75)
    at select.jsx:110:22
    at Option (select.jsx:109:108)
    at dev.js:515:12
    at untrack (dev.js:421:12)
    at Object.fn (dev.js:511:37)
```

Code:
```typescript
<Select options={['TODO', 'TODO2']} />
```
I can stub that out like this (`optionToValue` is also needed):
```typescript
<Select
  isOptionDisabled={() => false}
  optionToValue={(val) => val}
  options={['TODO', 'TODO2']}
/>
```
However at that point the component stops working, clicking on different select items doesn't change the value. It still fires the `onChange` event, but it doesn't update.

**Comment by martinpengellyphillips** [comment]
Confimed - looks like [Solid JS 1.6.0](https://github.com/solidjs/solid/releases/tag/v1.6.0) is the cause here as it works fine until upgrading to that version.

I'll chat to the community about what could be happening here.

**Comment by martinpengellyphillips** [comment]
Fixed in Solid JS [1.6.1](https://github.com/solidjs/solid/commit/763adf442c898adc4e6d6ae4a94e840338f17e7c) - please update to that version of Solid.
